src/server/socketHandlers/scGraphQLHandler.js

```diff
@@ -3,10 +3,9 @@ import Schema from 'server/graphql/rootSchema';
 
 // eslint-disable-next-line no-underscore-dangle
 const mutations = Schema._mutationType && Schema._mutationType._fields || {};
-const whitelist = ['updateUserProfile', 'summarizeMeeting'];
+const whitelist = ['createUserPicturePutUrl', 'updateUserProfile', 'summarizeMeeting'];
```

Comment:
yeah, i've been thinking cashay should just pass along a `context` to the server. Unfortunately, using cashay to do that requires logic at run time, which is unnecessary. we know what responses we want at build time, so why muddy up the runtime?